### ISSUE TITLE
Expose the X, Y, and Z magnitudes on EstimoteTelemetry

### ISSUE THREAD

**Issue by davslye** [Estimote/Android-SDK#221]
**TLDR:**
Expose/Make the X,Y,Z magnitudes on the [`Vector class`](http://estimote.github.io/Android-SDK/JavaDocs/com/estimote/sdk/telemetry/Vector.html) `public`.

**Desired Use case:**
In logging the acceleration from a connected Estimote, the independent magnitudes of the each axis (X,Y,Z) are not accessible as their primitive (nor boxed) values. Please expose the magnitudes of each axis as a primitive, or boxed type. The exposed magnitudes would be sufficient as read-only fields, and would not require setting permissions.

**Currently:**
Currently, the [accelerometer](http://estimote.github.io/Android-SDK/JavaDocs/com/estimote/sdk/telemetry/EstimoteTelemetry.html#accelerometer) on the [EstimoteTelemetry class](http://estimote.github.io/Android-SDK/JavaDocs/com/estimote/sdk/telemetry/EstimoteTelemetry.html) has a class type of [`Vector`](http://estimote.github.io/Android-SDK/JavaDocs/com/estimote/sdk/telemetry/Vector.html), which stores the XYZ axis magnitudes (and takes them in as a construction arguments!). However, the only way to access the axis' magnitude values _currently_ is to invoke [`EstimoteTelemetry.accelerometer.toString()`](http://estimote.github.io/Android-SDK/JavaDocs/com/estimote/sdk/telemetry/Vector.html#toString--) and parse the string into each axis' component values. This is inefficient.

Reference: 
[0] http://estimote.github.io/Android-SDK/JavaDocs/com/estimote/sdk/telemetry/EstimoteTelemetry.html#accelerometer
[1] http://estimote.github.io/Android-SDK/JavaDocs/com/estimote/sdk/telemetry/EstimoteTelemetry.html
[2] http://estimote.github.io/Android-SDK/JavaDocs/com/estimote/sdk/telemetry/Vector.html
[3] http://estimote.github.io/Android-SDK/JavaDocs/com/estimote/sdk/telemetry/Vector.html#toString--

**Comment by pawelDylag** [comment]
Hello @davslye, 

This problem was fixed since version `1.0.0` - [see the changelog](https://github.com/Estimote/Android-SDK/blob/master/CHANGELOG.md#100-march-29-2017).

Cheers!
